src/main/java/org/spongepowered/common/item/merchant/SpongeTradeOfferBuilder.java

```diff
@@ -68,6 +72,7 @@ public SpongeTradeOfferBuilder() {
 
     @Override
     public TradeOffer.Builder secondBuyingItem(final ItemStack item) {
+        Preconditions.checkNotNull(item, "Second buying item cannot be null");
```

Comment:
Can the second item not be null? I thought we allowed it to be null implicitly?

Reply:
The item can be empty.